plugins/woocommerce/tests/e2e-pw/tests/merchant/create-simple-product.spec.js

```diff
@@ -52,7 +52,9 @@ test.describe.serial( 'Add New Simple Product Page', () => {
 	} );
 
 	test( 'can create simple virtual product', async ( { page } ) => {
-		await page.goto( 'wp-admin/post-new.php?post_type=product' );
+		await page.goto( 'wp-admin/post-new.php?post_type=product', {
+			waitUntil: 'networkidle',
```

Comment:
Oh great spot, @alopezari! It seems that we should use `waitUntil` instead of `waitForLoadState`.

`waitUntil` is the valid option to use in `page.goto` as mentioned in the Playwright docs [here](https://playwright.dev/docs/api/class-page#page-goto-option-wait-until).

`waitForLoadState` however, is a function of `page` object, so `page.waitForLoadState` is still valid as mentioned in the Playwright docs [here](https://playwright.dev/docs/api/class-page#page-wait-for-load-state).

I created the task https://github.com/woocommerce/woocommerce/issues/36698 to change all occurrences of the invalid `waitForLoadState` option inside `page.goto()`. Do you think it's fine to address this change in a separate PR?

Reply:
Sure! Sounds good. Thanks for investigating it! 💯